iis-wf/iis-wf-import/src/test/java/eu/dnetlib/iis/wf/importer/converter/ProjectToOrganizationConverterTest.java

```diff
@@ -0,0 +1,168 @@
+package eu.dnetlib.iis.wf.importer.converter;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.util.Collection;
+import java.util.Iterator;
+import java.util.NavigableMap;
+import java.util.TreeMap;
+
+import org.apache.hadoop.hbase.client.Result;
+import org.junit.Test;
+
+import com.google.common.primitives.UnsignedBytes;
+
+import eu.dnetlib.data.proto.FieldTypeProtos.Qualifier;
+import eu.dnetlib.data.proto.KindProtos.Kind;
+import eu.dnetlib.data.proto.OafProtos.Oaf;
+import eu.dnetlib.data.proto.OafProtos.OafRel;
+import eu.dnetlib.data.proto.ProjectOrganizationProtos.ProjectOrganization;
+import eu.dnetlib.data.proto.ProjectOrganizationProtos.ProjectOrganization.Participation;
+import eu.dnetlib.data.proto.RelMetadataProtos.RelMetadata;
+import eu.dnetlib.data.proto.RelTypeProtos.RelType;
+import eu.dnetlib.data.proto.RelTypeProtos.SubRelType;
+import eu.dnetlib.iis.importer.schemas.ProjectToOrganization;
+import eu.dnetlib.iis.wf.importer.input.approver.ResultApprover;
+
+/**
+* @author mhorst
+*/
+
+public class ProjectToOrganizationConverterTest {
+
+	private byte[] relationColumnFamilyBytes = "dummy".getBytes();
+	
+    private Oaf oaf = null;
+    
+    private Result result = mock(Result.class);
+    
+    private ResultApprover defaultApprover = new ResultApprover() {
```

Comment:
It's much more advisable to use mocking here:
private ResultApprover defaultApprover = mock(ResultApprover.class).
Why? Think what happens if you need to add a new method to this interface.